app/helpers/ems_physical_infra_helper/textual_summary.rb

```diff
@@ -62,9 +62,13 @@ def textual_physical_servers
     h
   end
 
-  def textual_hosts
+  def textual_physical_servers_with_host
     count_of_host_relationships = (@record.physical_servers.select { |server| !server.host.nil? }).length
-    {:label => _("Hosts"), :icon => "pficon pficon-screen", :value => count_of_host_relationships}
+    h = {:label => _("Physical Servers with Host"), :icon => "pficon pficon-server", :value => count_of_host_relationships}
+    if count_of_host_relationships > 0
+      h[:link] = "/ems_physical_infra/#{@ems.id}?display=physical_servers&options=physical_servers_with_host"
```

Comment:
> can't we?

We totally can! :)

The model lookup was happening here -
```
~/manageiq-ui-classic/app/helpers/application_helper.rb:248:in `model_string_to_constant'
~/manageiq-ui-classic/app/controllers/application_controller.rb:331:in `process_params_model_view'
~/manageiq-ui-classic/app/controllers/application_controller.rb:368:in `report_data'
```

and after adding the `"physical_servers_with_host"` in `MODEL_STRING`, the list renders properly.

```ruby
MODEL_STRING = {
    "all_vms"           => VmOrTemplate,
    "all_miq_templates" => MiqTemplate,
    "instances"         => Vm,
    "images"            => MiqTemplate,
    "groups"            => Account,
    "users"             => Account,
    "host_services"     => SystemService,
    "chargebacks"       => ChargebackRate,
    "physical_servers_with_host" => PhysicalServer,
  }.freeze

```

Thanks for the tip.